app/app/src/main/res/values/joystick_attri.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think it would be better to have a single "attrs.xml" file than to have one for every element, in case we create more attributes for different canvases and custom views. It should be possible to have more than one "declare-stylable" tag in a xml file.

Reply:
To answer your question on slack @ZJW-92 : What I mean is that the attrs file you created is named "joystick_attri.xml" instead of just "attrs.xml", which means that it's joystick specific.
Attributes can be added for other elements in our app as well, and it would be better for scalability to store them all in one file rather than having multiple attribute xml files.